components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/response/OAuthAPIResponse.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2.token.handlers.response;
+
+import org.apache.oltu.oauth2.as.response.OAuthASResponse;
+
+/**
+ * Holds response for the OAuth flow api calls.
+ */
+public class OAuthAPIResponse extends org.apache.oltu.oauth2.as.response.OAuthASResponse {
```

Comment:
How about a name like OAuth2TokenResponse or something?
OAuthAPIResponse is a bit ambigious unless this class can be used in both authorize and token flows

Reply:
+1. Fixed with fcc459bddf38ae1fb01650ec7c57acea167ad23c